meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* JZ: Is there no way to convince them to participate in CG?
+* eP: you say that CG decisions through pivot to solidcommunity.net become de-facto standards, but I don't see it that way, and it's misinforming people. What the CG does have zero standing until the WG adopts it. We can say we avoid making changes without technical commitment, but the most honest way is to say our work has no formal standing.
+* MdJ: I don't think we will reach conclusion or solution. Probably we can reach PR 711 and be aware of the dynamics.
+* T: ... unstable solidcommunity.net
+* JZ: There is already testing insteance.
+* MdJ: What does it mean that something gets merged in the spec repo. Someone runs into an issue using solidcommunity.net. Makes a PR to the specification repo, it gets merged, deployed to solidcommunity.net. If everyone runs their own server coherence gets even lower. Solidcommunity.net is used by SolidOS team to do demos etc. If we don't update that server we may don't bother updating the specification repo. 
+* eP: Even though the repo is called 'specification', we could include some more inline notes in cases where there is no full concensus.
+* MdJ: This way we can't have a test suite. We can't have compatible implementations. We already have some servers doing WAC, some ACP, some SAI others do UMA. 
+* MdJ: If Solid is a way to signal what can be done but there is no consistent way of achieving it. ...
+* JW: The CG is here to incubate specs. It should be a goal of consensus rather than specification divergance. For issue 711, that was particularly controversial, I don't think we will hit that many issues with this degree of controversy. 
+* MdJ: WAC vs. ACP, SAI, UMA.
+* JW: for those, my view that CG should be incubating both WAC and ACP. We can reach consensus and LWS WG can look at them and decide what will be put forward in LWS spec and have more contancious debate there. For SAI vs. Type indexes etc. Specs that are not in scope of LWS, we need to decide how to choose from them. This should be done in data driven maner. Seeing how the cover the functional requirements. Data interop spec should be one 
+* eP: For example with SAI it's a little bit more complex, it's not SAI vs type indexes, there is also quite a lot about authorization there, and for the LWS WG there are VCs and data grants in scope, so there is some overlap there.
+* eP: I'm happy that JW refers to requirements, like the WG, the CG needs to work with scopes and requirements. The conversations often seem to be emotional and sentimental. But we should look at which requirements each proposal satisfies.
+* JW: The data grants, I belive are mentikoned in LWS charter. In ESS consent flow VCs are used as part of access requrest mechanism. That has been put into ACP via `acp:vc`. I would have a conversation with Aaron as well as Wouter who is working on UMA flow for CSS. You could see if you can come to rought consensus. Otherwise this can split into different specs.
```

Comment:
```suggestion
* JW: I believe data grants are mentioned in LWS charter. In ESS, consent flow VCs are used as part of the access request mechanism. That has been put into ACP via `acp:vc`. I would have a conversation with Aaron as well as Wouter, who is working on UMA flow for CSS. You could see if you can come to rough consensus. Otherwise, this can split into different specs.
```